matmul/rust-ndarray/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "rust-ndarray"
+version = "0.0.1"
+edition = "2021"
+
+[profile.release]
+lto = true
+codegen-units = 1
+panic = "abort"
+
+[dependencies]
+ndarray = "0.15"
```

Comment:
Please use the most specific version as possible, so it's easier to track the changes and versions. Right now, it's "0.15.4".

Another note here is that you use the default feature set that could be not the ideal for performance tests. BLAS integration and other features could be a game changer. I'd leave it to you to experiment here.

Also, please add a newline at the end of the file (`git` usually warns about it too).